ruby/ex4.rb

```diff
@@ -0,0 +1,21 @@
+class SportsHandler
+    def skip_sports(sports, skip)
+      result = []
+      # In Ruby, the each_with_index method is used to iterate over a collection (such as an array or enumerable) while keeping track of the index of the current element
+      sports.each_with_index do |sport, index|
+        if index >= skip
+          result << "#{index + 1}:#{sport}"
+        end
+      end
+    
```

Comment:
could have handled the error for out of bounds instead of returning a null array